.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v2
```

Comment:
Q: Gibt es hier neuere Versionen der `setup-dotnet`-Action? Falls ja, bitte die neuste Version verwenden und auch die anderen Actions auf Updates prüfen. Kann in einem separaten PR erfolgen.

Reply:
Ich habe Issue #332 dazu erstellt und werde die Actions in einem separaten PR aktualisieren.